css/components/button.css

```diff
@@ -0,0 +1,96 @@
+.btn {
```

Comment:
Hey, great job !! but I suggest that you give rounded corners to your buttons or any component that has a sharp edge. due to the following reasons:
Rounded Corners Make Information Easier to Process
Rounded corners are more effective for maps and diagrams because they allow our eyes to easily follow lines “as it suits better to the natural movement of the head and eyes respectively” . Sharp corners throw your eyes off the path of the line so you end up experiencing abrupt pauses when the line changes direction. 

Read more about it here:
https://uxmovement.com/thinking/why-rounded-corners-are-easier-on-the-eyes/#:~:text=Rounded%20corners%20are%20more%20effective,when%20the%20line%20changes%20direction.

Reply:
Thanks for the feedback, buttons were rounded initially, thought it was inconsistent so changed it, will change it again

Reply:
glad I was help !!